docs/rules/no-extra-parens.md

```diff
@@ -22,6 +22,7 @@ This rule has an object option for exceptions to the `"all"` option:
 * `"returnAssign": false` allows extra parentheses around assignments in `return` statements
 * `"nestedBinaryExpressions": false` allows extra parentheses in nested binary expressions
 * `"ignoreJSX": "none|all|multi-line|single-line"` allows extra parentheses around no/all/multi-line/single-line JSX components. Defaults to `none`.
+* `"enforceForArrowConditionals": false` allows extra parentheses around arrow conditionals expression
```

Comment:
Can you be more descriptive here? For example, "allows extra parentheses around ternary expressions which are the body of an arrow function"